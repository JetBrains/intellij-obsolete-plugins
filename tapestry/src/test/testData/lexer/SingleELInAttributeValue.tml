<t:hidden value="${rrrrr?.tt.r(77, -5.3, [-3,
     -2])}"><title>Hello World Page</title></t:hidden>