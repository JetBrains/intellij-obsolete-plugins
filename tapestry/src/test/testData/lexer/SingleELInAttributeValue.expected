XML_START_TAG_START ('<')
XML_NAME ('t:hidden')
WHITE_SPACE (' ')
XML_NAME ('value')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
TAP5_EL_HOLDER ('${rrrrr?.tt.r(77, -5.3, [-3,
     -2])}')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_START_TAG_START ('<')
XML_NAME ('title')
XML_TAG_END ('>')
XML_DATA_CHARACTERS ('Hello')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('World')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('Page')
XML_END_TAG_START ('</')
XML_NAME ('title')
XML_TAG_END ('>')
XML_END_TAG_START ('</')
XML_NAME ('t:hidden')
XML_TAG_END ('>')
